synapse/cortex.py

```diff
@@ -538,15 +537,14 @@ class Cortex(s_cell.Cell):
 
         await self.addCoreMods(s_modules.coremods)
 
+        self.triggers = s_trigger.Triggers(self)
+
         mods = self.conf.get('modules')
 
         self._initFormCounts()
 
         await self.addCoreMods(mods)
 
-        if self.conf.get('triggers:enable'):
```

Comment:
If we're removing the abiltity to disable triggers, we need to remove the trigger configuration option.